app/src/main/java/com/yalta/activities/MainActivity.kt

```diff
@@ -10,24 +10,50 @@ import com.yalta.activities.fragments.MapFragment
 import com.yalta.activities.fragments.ProfileFragment
 
 class MainActivity : AppCompatActivity() {
+    private val browseFragment = BrowseFragment()
+    private val mapFragment = MapFragment()
+    private val profileFragment = ProfileFragment()
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
         val navigation: BottomNavigationView = findViewById(R.id.bottom_navigation)
 
+        supportFragmentManager.beginTransaction()
+            .add(R.id.container, browseFragment)
+            .add(R.id.container, mapFragment)
+            .hide(mapFragment)
+            .add(R.id.container, profileFragment)
```

Comment:
weird order

Reply:
Fragments are added in order they are placed on navigation bar
And I've decided to hide them where they're added just not to forget anything